pyvista/plotting/lookup_table.py

```diff
@@ -199,7 +201,7 @@ class LookupTable(_vtk.vtkLookupTable):
     """
 
     _nan_color_set = False
-    _cmap = None
+    _cmap: colors.Colormap | colors.ListedColormap | None = None
```

Comment:
There's a broken link for `colors.ListedColormap` :
```
Build error: `/pyvista/plotting/lookup_table.py:docstring of pyvista.LookupTable.cmap:1: WARNING: py:class reference target not found: colors.ListedColormap [ref.class]
```

Might need to add it to `nitpick_regex_ignore` in the docs `conf.py`

Reply:
Thanks , done in https://github.com/pyvista/pyvista/commit/bcdbac94fa277e1e647d3d9432034ae5f4aafbe9